.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,93 @@
+name: "Bug report"
+description: "Create a report to help us improve TypeScript"
+title: "[BUG] <description>"
+body:
+  - type: markdown
+    attributes:
+      value: Please fill in each section completely. Thank you!
+  - type: input
+    id: search_terms
+    attributes:
+      label: "🔎 Search Terms"
+      description:
+        "What search terms did you use when trying to find an existing bug report?
+
+        List them here so people in the future can find this one more easily."
+    validations:
+      required: true
+  - type: textarea
+    id: version_info
+    attributes:
+      label: "🕗 Version & Regression Information"
+      description: "
```

Comment:
Can this just be a `|` string rather than having to quote things?

Similarly for all other multi-line strings like this.